frontend/src/app/components/transaction/transaction.component.html

```diff
@@ -8,9 +8,9 @@
         <span class="d-none d-lg-inline">{{ rbfTransaction.txid }}</span>
       </a>
     </div>
-  
+
     <div>
-      <div class="title float-left">
```

Comment:
We should keep this to maintain the RTL hint

Reply:
I've solve this by adding the classes under the `.rtl-layout`.